packages/data-stores/src/queries/use-all-domains-query.ts

```diff
@@ -25,7 +25,10 @@ export function useAllDomainsQuery( options: UseQueryOptions< AllDomainsQueryFnD
 	return useQuery( {
 		queryKey: [ 'all-domains' ],
 		queryFn: () =>
-			wpcomRequest< AllDomainsQueryFnData >( { path: '/all-domains', apiVersion: '1.1' } ),
+			wpcomRequest< AllDomainsQueryFnData >( {
+				path: '/all-domains?test_data=true&domain_count=2',
```

Comment:
Do we want to merge the test data params or just allow devs to add it when they need test data?

Reply:
Oops!

Reply:
Removed ✅